runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -370,6 +372,32 @@ public MouseEvent mousePressed(MouseEvent mouseEvent)
 
 					return mouseEvent;
 				}
+
+				@Override
+				public MouseEvent mouseMoved(MouseEvent mouseEvent)
+				{
+					if (!(ClientUI.this.client instanceof Client) || withTitleBar)
+					{
+						return mouseEvent;
+					}
+
+					if (sidebarButtonRange.contains(mouseEvent.getPoint()))
+					{
+						mouseInSidebarButtonRange = true;
+					}
+					else if (mouseInSidebarButtonRange)
+					{
+						mouseInSidebarButtonRange = false;
+
+						final Client client = (Client) ClientUI.this.client;
+						//Repaint of client canvas required to remove image artifact in black bars when on login screen with stretched enabled.
+						if (client.getGameState() == GameState.LOGIN_SCREEN)
+						{
+							client.getCanvas().repaint();
```

Comment:
Is this really necessary? I think this is general stretched mode bug and we should not be solving it case-per-case

Reply:
If I removed it the sidebar toggle button won't disappear on the login screen when the mouse exits the "area", but I guess that's not too big an issue as the main desired effect is hiding it when in game, so I can remove this if you want.

Reply:
Yeah, that would greatly simplify the code (as the entire mouse moved and boolean would not be necessary)